sql/mysql/Positive-Technologies/MySqlParser.g4

```diff
@@ -350,6 +350,8 @@ columnDefinition
 columnConstraint
     : nullNotnull                                                   #nullColumnConstraint
     | DEFAULT defaultValue                                          #defaultColumnConstraint
+    | VISIBLE                                                       #visibleColumnConstraint
+    | INVISIBLE                                                     #invisibleColumnConstraint
```

Comment:
FYI, It's possible to use the same label here `visibilityColumnConstraint` if it's convenient for you:

```g4
    | VISIBLE                                                       #visibilityColumnConstraint
    | INVISIBLE                                                     #visibilityColumnConstraint
```

ANTLR merges such labels into a single alternative.

Reply:
sure, updated.